internal/gateway/proxy/proxy_test.go

```diff
@@ -250,3 +256,129 @@ func TestControllerProxy_LookLookupOffset(t *testing.T) {
 
 	})
 }
+
+func TestControllerProxy_ValidateSubscription(t *testing.T) {
+	Convey("test ValidateSubscription", t, func() {
+		cp := NewControllerProxy(Config{
+			Endpoints: []string{"127.0.0.1:20001",
+				"127.0.0.1:20002", "127.0.0.1:20003"},
+			CloudEventReceiverPort: 18080,
+			ProxyPort:              18082,
+			Credentials:            insecure.NewCredentials(),
+		})
+
+		data := `{
+    "id":"13b719a4-ada9-436a-9fb1-fc2bc82dc647",
+    "source":"prometheus",
+    "specversion":"1.0",
+    "type":"naive-http-request",
+    "datacontenttype":"application/json",
+    "subject":"operator",
+    "time":"2022-12-12T08:31:54.936803649Z",
+    "data":{"body":{"alerts":[{"annotations":{"feishuUrls":[{"URL":"https://open.feishu.cn/open-apis/bot/v2/hook/xxxxx",
```

Comment:
It‘s not better to use user example data

Reply:
I have deleted sensitive data, structure is ok because it made by Prometheus